editor/components/post-permalink/index.js

```diff
@@ -123,7 +123,13 @@ class PostPermalink extends Component {
 
 export default compose( [
 	withSelect( ( select ) => {
-		const { isEditedPostNew, isPermalinkEditable, getEditedPostPreviewLink, getPermalink, isCurrentPostPublished } = select( 'core/editor' );
+		const {
+			isEditedPostNew,
+			isPermalinkEditable,
+			getEditedPostPreviewLink,
```

Comment:
Noted above: `getEditedPostPreviewLink` is a useless selector, should be removed here (and altogether, since this is its only remaining usage).